### ISSUE TITLE
Features: Color Selection, Flash, Rainbow Mode

### ISSUE THREAD

**Issue by GrumpyTanker** [DutchDevelop/BLLEDController#7]
Just ordered one of these from etsy, really looking forward to using it. Had a few suggestions to increase the functionality.

Saw that you implemented brightness control, so it'd be nice to be able to set different colors for the various faults instead of them all being red.

Another nice feature would be being able to flash or cycle between two colors. Again, this would be nice for various faults to differentiate. That way red still means a problem, but then the 2nd color would indicate what type of problem.

Rainbow mode to cycle through the rainbow colors would just be cool for all us RGB fanboys out here. Might be a nice light show for when the printer is idle or successfully completed a print.

**Comment by bolsoncerrado** [comment]
Indeed, differentiating between simple "filament spool empty" or "any other major issue" would be great  ;) I second it.
